sudachi/src/plugin/connect_cost.rs

```diff
@@ -23,7 +23,7 @@ use crate::dic::grammar::Grammar;
 use crate::prelude::*;
 
 /// Trait of plugin to edit connection cost in the grammar
-pub trait EditConnectionCostPlugin: Sync {
+pub trait EditConnectionCostPlugin: Sync + Send {
```

Comment:
Can we merge adding `Send` to plugins before the full Python integration? I will need to propagate this change to plugin loader/holder rewrite as well.

Reply:
Yes, I'll separate this part.